R/metadata.R

```diff
@@ -51,5 +54,8 @@ xportr_metadata <- function(.df, metadata = NULL, domain = NULL) {
 
   ## End of common section
 
-  structure(.df, `_xportr.df_metadata_` = metadata)
+  structure(.df,
```

Comment:
Very very optional. I would think that {styler} or {lintr} wouldn't like this indentation

```suggestion
  structure(
    .df,
```